types/int.go

```diff
@@ -422,29 +422,39 @@ func (i Uint) AddRaw(i2 uint64) Uint {
 // Sub subtracts Uint from another
 func (i Uint) Sub(i2 Uint) (res Uint) {
 	res = Uint{sub(i.i, i2.i)}
-	// Check overflow
-	if res.Sign() == -1 || res.Sign() == 1 && res.i.BitLen() > 256 {
+	if UintOverflow(res) {
 		panic("Uint overflow")
 	}
 	return
 }
 
+// SafeSub attempts to subtract one Uint from another. A boolean is also returned
+// indicating if the result contains integer overflow.
+func (i Uint) SafeSub(i2 Uint) (Uint, bool) {
```

Comment:
can we use the same name for consistency (either `Minus` or `Sub`)?

Reply:
same with `Plus` and `Add`

Reply:
Yes, this was annoying to me during the changes. But while I agree with you, I think this should be done in a separate PR. Thoughts?

Reply:
I prefer Add and Sub for both.  Lets do a separate PR.

Reply:
++

Will make an issue.